common/persistence/domainReplicationQueue.go

```diff
@@ -110,18 +116,63 @@ func (q *domainReplicationQueueImpl) UpdateAckLevel(lastProcessedMessageID int,
 		return fmt.Errorf("failed to update ack level: %v", err)
 	}
 
-	select {
-	case q.ackNotificationChan <- true:
```

Comment:
is this channel not consumed anywhere before ? I wonder what the original intent was

Reply:
It used in the original design. I am adding it back